spec/rubocop/cop/style/string_concatenation_spec.rb

```diff
@@ -52,6 +52,25 @@
     RUBY
   end
 
+  context 'implicit concatenation' do
+    it 'registers an offense and corrects for string concatenation' do
+      expect_offense(<<~RUBY)
+        "a" + "b" "c"
+        ^^^^^^^^^^^^^ Prefer string interpolation to string concatenation.
+        "a" "b" + "c"
+        ^^^^^^^^^^^^^ Prefer string interpolation to string concatenation.
+        "a" + "b" "c" + "d"
+        ^^^^^^^^^^^^^^^^^^^ Prefer string interpolation to string concatenation.
```

Comment:
Can you split each of them into three separate tests?

Reply:
Done.